{{cookiecutter.repo_name}}/styles/main.scss

```diff
@@ -1,18 +1,33 @@
 /**
- * {{cookiecutter.project_name}}
+ * Ethanol
```

Comment:
Minor thing here, we should probably keep the `{{cookiecutter.project_name` here. :)